packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -255,6 +280,16 @@ export default class QuanticSort extends LightningElement {
     return this.hasInitializationError || !!this.errorMessage;
   }
 
+  get sortContainerCSSClass() {
+    return this.variant === SORT_VARIANTS.DEFAULT
+      ? 'sort__container'
+      : undefined;
+  }
```

Comment:
I'd already commented on that, don't use undefined on purpose most of the time it's a bad idea.

Pretty sure Salesforce would render this component as `<my-component class="undefined"></my-component>` if you put `undefined` there.

```suggestion
  get sortContainerCSSClass() {
    return this.variant === SORT_VARIANTS.DEFAULT
      ? 'sort__container'
      : '';
  }
```

Reply:
hmm yeah maybe, but I realized I actually don't need that anymore so I just removed the getter altogether